test/SecTester.Scan.Tests/ScanTests.cs

```diff
@@ -1,53 +1,228 @@
+using Microsoft.Extensions.Logging;
+using NSubstitute.ClearExtensions;
+using SecTester.Scan.Tests.Fixtures;
+
 namespace SecTester.Scan.Tests;
 
-public class ScanTests
+public class ScanTests : ScanFixture
 {
+  private readonly Scans _scans = Substitute.For<Scans>();
+  private readonly LoggerMock _logger = Substitute.For<LoggerMock>();
+
+  public override void Dispose()
+  {
+    _scans.ClearSubstitute();
+    _logger.ClearSubstitute();
+
+    base.Dispose();
+  }
+
+  [Fact]
+  public void Constructor_GivenNullId_ThrowError()
+  {
+    // act
+    Action act = () => new Scan(null!, _scans, _logger, new ScanOptions());
```

Comment:
Use var instead (the same in other places)

```suggestion
    var act = () => new Scan(null!, _scans, _logger, new ScanOptions());
```